packages/camera/tizen/src/camera_device.cc

```diff
@@ -993,6 +993,11 @@ void CameraDevice::Open(
           self->prepared_packet_ = packet;
           return;
         }
+        if (self->is_preview_paused_) {
+          media_packet_destroy(packet);
+          self->prepared_packet_ = nullptr;
+          return;
+        }
```

Comment:
Simply discard the prepared packet when `is_preview_paused_` is true, without stopping the preview itself (`camera_stop_preview`).

How do you like this approach? @bbrto21

Reply:
Great! but the relationship between `Is_preview_paused_` and `StopCameraPreview` seems ambiguous. how about adding some comments?

Remark: Preview affects the state of the camera device. also it always pause the preview before start recording using `StopCameraPreview`

Reply:
If I can safely call `StopCameraPreview` and `StartCameraPreview` at any time and it makes more sense to you, I'll change the implementation.

Reply:
I'm totally fine with current Implementation. but I'll check the safety when I get back to the office :)

Reply:
I tested [this change](https://github.com/swift-kim/plugins/commit/fd32be23507dddfbc7c80becefb4d7202bf95bc4) on a TM1 device today, but it didn't work as expected, because the capture API requires the camera state to be `CAMERA_STATE_PREVIEW` before capturing. Even if I call `StartCameraPreview` before calling `StartCameraCapture` and call `StopCameraPreview` again after the capture is finished, the call to `StopCameraPreview` results in an `Invalid state` error for unknown reason.

The problem of the current implementation is that the preview textures are not delivered to Flutter even when the user is recording a video. This is inconsistent with the Android side implementation. Let's revisit this issue when it's needed.